src/transformers/onnx/features.py

```diff
@@ -34,7 +34,7 @@
         AutoModelForSequenceClassification,
         AutoModelForTokenClassification,
     )
-elif is_tf_available():
+elif is_tf_available() and not is_torch_available():
```

Comment:
I think the whole logic of having three tests can be simplified if you just change that `elif` to a simple `if`.